analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LockedFieldShouldBeReadonly.CSharp8.cs

```diff
@@ -0,0 +1,46 @@
+﻿class Test
+{
+    static readonly object staticReadonlyField = null;
+    static object staticReadWriteField = null;
```

Comment:
Same for this - it can be deleted.

Reply:
`staticReadWriteField` seems the only one which is not used anymore:
- `staticReadonlyField` reference is returned on line 29
- `readonlyField` reference is returned on line 14
- `readWriteField` reference is returned on line 17

@pavel-mikula-sonarsource: was it rather 3 out of 4 to be kept?